services/consensus_create_topic.proto

```diff
@@ -30,44 +37,101 @@ import "basic_types.proto";
 import "duration.proto";
 
 /**
- * See [ConsensusService.createTopic()](#proto.ConsensusService)
+ * Create a topic to accept and group consensus messages.
+ *
+ * If `autoRenewAccount` is specified, that account Key MUST also sign this
+ * transaction.<br/>
+ * If `adminKey` is set, that Key MUST sign the transaction.<br/>
+ * On success, the resulting `TransactionReceipt` SHALL contain the newly
+ * created `TopicId`.
+ *
+ * The `autoRenewPeriod` on a topic MUST be set to a value between
+ * `MIN_AUTORENEW_PERIOD` (6999999 seconds/81 days) and
```

Comment:
I did find this checking on topicCreate.

Reply:
Will Fix.